google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/GctFeature.java

```diff
@@ -24,16 +24,34 @@
 
 /**
  * The set of Google Cloud Tools features that we want to be able to conditionally enable.
+ *
+ * <p>Enum elements marked {@link Deprecated} are fully released and no longer hidden behind a flag.
  */
 public enum GctFeature implements Feature {
+  @Deprecated
   APPENGINE_FLEX(null, "feature.appengine.flex", "ct4ij.feature.appengine.flex");
```

Comment:
Ack. Could be good to add a clarifying "// Left here just to serve as an example. Don't use." comment. I think that's why you added the @Deprecated annotation for.